client/src/components/Navbar/Navbar.js

```diff
@@ -0,0 +1,107 @@
+/** @format */
```

Comment:
similar to other comments

Reply:
I use prettier. when I save it auto came up.